src/Compilers/CSharp/CscCore/project.json

```diff
@@ -1,42 +1,45 @@
 {
   "dependencies": {
-    "System.AppContext": "4.0.0-beta-22816",
```

Comment:
The vbccore project.json was not the same as the csccore project.json. I consulted with @jaredpar and he said they should be the same and the C# version looked more correct. So I'll copy the C# version to the VB version.